allennlp/modules/layer_norm.py

```diff
@@ -0,0 +1,25 @@
+
+import torch
+
+
+class LayerNorm(torch.nn.Module):
```

Comment:
Should this be `LayerNormalization`?  I could go either way here.  "norm" is pretty standard.